FirebaseAdmin/FirebaseAdmin.Tests/Auth/UtilTest.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021, Google Inc. All rights reserved.
```

Comment:
UtilsTest

Reply:
> Can we incorporate this change into this PR?

Absolutely, I'll  get right on it

Reply:
As I was implementing it I noticed I could simplify the FirebaseUserManager constructor further by letting the GetIdToolkitHost function handle tenant ids as well. https://github.com/firebase/firebase-admin-dotnet/pull/289/commits/c190edc8fd89219d0f1a2dc302367b219ffc7fad

The GoogleCredentials resolution: https://github.com/firebase/firebase-admin-dotnet/pull/289/commits/31ec0635ff2523b0614c3424e74e2b936f21c1ef